mico-core/src/main/java/io/github/ust/mico/core/REST/ApplicationController.java

```diff
@@ -320,4 +322,26 @@ private URI getPrometheusUri(String query, String podName) {
             throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "There is no such application/service");
         }
     }
+
+    /**
+     * Returns a list of services associated with the mico application specified by the parameters.
+     * @param applicationShortName
+     * @param applicationVersion
+     * @return
+     */
+    @GetMapping("/{" + PATH_VARIABLE_SHORT_NAME + "}/{" + PATH_VARIABLE_VERSION + "}/" + PATH_SERVICES)
+    public ResponseEntity<Resources<Resource<MicoService>>> getMicoServicesFromApplication(@PathVariable(PATH_VARIABLE_SHORT_NAME) String applicationShortName,
+                                                                                           @PathVariable(PATH_VARIABLE_VERSION) String applicationVersion) {
+        Optional<MicoApplication> applicationOptional = applicationRepository.findByShortNameAndVersion(applicationShortName, applicationVersion);
+        if (applicationOptional.isPresent()) {
```

Comment:
Move fail condition to the top. This also gets rid of one indention level for the actual code.